gulp/tasks/webpack.js

```diff
@@ -12,46 +12,52 @@ function webpack() {
     const plugins = isDevelopment ?
         [new HardSourceWebpackPlugin()] :
         [
-            new webpack2.optimize.UglifyJsPlugin(),
+            new webpack2.optimize.UglifyJsPlugin({ sourceMap: true }),
             new webpack2.optimize.MinChunkSizePlugin({minChunkSize: 16000}),
         ];
 
     return gulp.src([
         `${config.dest}/app/main.js`
-    ])
-    .pipe(webpackStream({
-        // watch: true, // This causes gulp to freeze and not serve
-        output: {
-            path: path.resolve(config.dest),
-            filename: "bundle.js",
-            publicPath: "/", // for where to request chunks when the SinglePageApp changes the URL
-            chunkFilename: "chunk-[chunkhash].js"
-        },
-        externals: {
-            settings: 'SETTINGS'
-        },
-        module: {
-            rules: [
-                {
-                    test: /\.css$/,
-                    loader: 'ignore-loader'
-                },
-                {
-                    test: /\.map$/,
-                    loader: 'ignore-loader'
-                }
-            ]
-        },
-        plugins,
-        resolve: {
-            alias: {
-                "settings": path.resolve(config.dest, "settings.js"),
-                "~": path.resolve(config.dest, "app/"),
-            }
-        },
-        devtool: 'source-map'
+    ]).pipe(webpackStream({
+      // watch: true, // This causes gulp to freeze and not serve
+      externals: {
+          settings: {
```

Comment:
I think this is the source of the settings problem. Is there a reason it shouldn't be `settings: 'SETTINGS'`?
Setting it to that let the page load, but no CSS is loading.

Reply:
ah yep. I think I started the branch before master had the `SETTINGS` changes and I added them manually to just get something to build. Fixed!